tests/scenarios/delegated-grant.spec.ts

```diff
@@ -227,6 +226,63 @@ export function testDelegatedGrantScenarios(): void {
       const participantRoleReply = await dwn.processMessage(bob.did, participantRoleRecord.message, { dataStream: participantRoleRecord.dataStream });
       expect(participantRoleReply.status.code).to.equal(202);
 
+      // Alice creates a delegated subscribe grant for device X to act as Alice.
```

Comment:
Need to minimally update the title and scenario summary, but due to the nature of difference in sequencing, I feel like this is probably better off being it's own scenario.

Reply:
Yeah I was thinking about that as well but didn't want to have too much duplication.

But if you also feel that a separate scenario wouldn't be bad I'll break it out into it's own.

Reply:
Done, as well as in the `should fail if delegated grant has a mismatching protocol scope - subscribe`